config/locales/pt-BR/pt-BR.yml

```diff
@@ -319,6 +319,8 @@ pt-BR:
       show:
         edit: "Editar Repositório"
         destroy: "Remover Repositório"
+      edit:
+        edit: "Editar"
```

Comment:
If you use the translation present in the views.yml, there is no need to create another one here. 
```suggestion
```